pkgs/by-name/im/impala/package.nix

```diff
@@ -0,0 +1,25 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "impala";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "pythops";
+    repo = "impala";
+    rev = "v${version}";
+    hash = "sha256-r/aWzSn/Dci69oS/yopG6Ro34U8hniHVanctyM7RvDw=";
+  };
+
+  cargoHash = "sha256-IV1ftsRyM0CUlQMVmLip1FiqnouT5TsKASpF/KLARqY=";
+
+  meta = with lib; {
+    description = "TUI for managing wifi";
+    homepage = "https://github.com/pythops/impala";
+    license = licenses.gpl3Only;
+    maintainers = [ maintainers.nydragon ];
```

Comment:
It would be great to restrict platforms to the same  `iwd` supports

Reply:
Addressed the suggestion!